blockchains/smartchain/tokenlist.json

```diff
@@ -253,6 +256,19 @@
       "logoURI":"https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/smartchain/assets/0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3/logo.png",
       "pairs":[
 
+      ]
+    },
+    {
+      "chainId":56,
+      "asset":"c20000714_t0xD98560689C6e748DC37bc410B4d3096B1aA3D8C2",
```

Comment:
remove this too. There is no liquidity for the token

Reply:
Fixed